addons/im_livechat/static/src/composer/composer_patch.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="im_livechat.Composer" t-inherit="mail.Composer" t-inherit-mode="extension" owl="1">
+        <xpath expr="//*[hasclass('o-mail-Composer-coreHeader')]" position="inside">
+            <span t-if="thread?.type === 'livechat'" class="text-muted form-text">Press tab to go to the oldest unread thread</span>
+        </xpath>
+    </t>
+</templates>
```

Comment:
Should not show message if there is not oldest unread thread.
Also would make the following UI improvements:
- text in italics + footer + below "Send" so it's less distracting
- more concise text: _"Tab to next livechat"_
This will look like this:
<img width="1213" alt="Screenshot 2023-06-26 at 11 37 49" src="https://github.com/odoo/odoo/assets/6569390/5ac6c74f-79db-4575-af77-5418b7070f5f">

I would also not show this text in chat window, because this is far too noisy:
<img width="1100" alt="Screenshot 2023-06-26 at 11 44 11" src="https://github.com/odoo/odoo/assets/6569390/79c78dcf-1b22-4108-ae3f-594aea0ad036">
I'm not entirely sure we want this feature with chat windows. I would play safe and enable it only in discuss app, as there's a clear text hint there and we won't have to solve UI issues in chat windows if it's specific to discuss app.

If we really want to have it in chat windows, we should take more time to think about UI, and also a way to permanently close this hint.